ApplicationCode/GeoMech/GeoMechDataModel/RigFemPartResultCalculatorPoreCompressibility.cpp

```diff
@@ -210,52 +214,58 @@ RigFemScalarResultFrames*
 
                             int nodeIdx = femPart->nodeIdxFromElementNodeResultIdx( elmNodResIdx );
 
-                            // Calculate bulk modulus for solids (grains)
+                            // Calculate bulk modulus for solids (grains).
+                            // Incoming unit for Young's Modulus is GPa: convert to Pa.
                             double poissonRatio     = poissonRatioData[elmIdx];
-                            double youngsModuli     = youngsModuliData[elmIdx];
+                            double youngsModuli     = youngsModuliData[elmIdx] * 1.0e9;
                             double bulkModulusFrame = youngsModuli / ( 3.0 * ( 1.0 - 2.0 * poissonRatio ) );
                             double bulkModulus      = bulkModulusFrame / ( 1.0 - biotCoefficient );
 
                             // Calculate porosity
                             double voidr    = voidRatioData[elmNodResIdx];
                             double porosity = voidr / ( 1.0 + voidr );
 
-                            // Calculate difference in pore pressure between reference state and this state
-                            double initialPorePressure = initialPorFrameData[nodeIdx];
-                            double framePorePressure   = porFrameData[nodeIdx];
-                            double deltaPorePressure   = framePorePressure - initialPorePressure;
+                            // Calculate difference in pore pressure between reference state and this state,
+                            // and convert unit from Bar to Pascal.
+                            double referencePorePressure = referencePorFrameData[nodeIdx];
+                            double framePorePressure     = porFrameData[nodeIdx];
+                            double deltaPorePressure     = ( framePorePressure - referencePorePressure ) * 100000.0;
```

Comment:
Move unit conversion factors into RiaGeoMechDefines and name ie BarToPascal

Reply:
Fixed.